CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # History
 
+## 5.0.0
+
+- Node versions greater than 8 are supported.
+- Changes to vocabulary use:
```

Comment:
Maybe also introduce the new predicates we've added, such as `solid:storageQuota`?

Reply:
Mmmm, yeah, but that was actually earlier. We need to document it, yes. Perhaps on master...